cloudbuild-tag.yaml

```diff
@@ -8,23 +8,51 @@ steps:
+    docker pull gcr.io/$_STAGING_PROJECT_ID/k8s/deployer_helm/onbuild:sha_$COMMIT_SHA
+    docker tag gcr.io/$_STAGING_PROJECT_ID/k8s/deployer_helm/onbuild:sha_$COMMIT_SHA gcr.io/$PROJECT_ID/k8s/deployer_helm/onbuild:$TAG_NAME
 
-    docker pull gcr.io/marketplace-k8s-app-tools/k8s/deployer_helm_tiller:sha_$COMMIT_SHA
-    docker tag gcr.io/marketplace-k8s-app-tools/k8s/deployer_helm_tiller:sha_$COMMIT_SHA gcr.io/$PROJECT_ID/k8s/deployer_helm_tiller:$TAG_NAME
+    docker pull gcr.io/$_STAGING_PROJECT_ID/k8s/deployer_helm_tiller:sha_$COMMIT_SHA
+    docker tag gcr.io/$_STAGING_PROJECT_ID/k8s/deployer_helm_tiller:sha_$COMMIT_SHA gcr.io/$PROJECT_ID/k8s/deployer_helm_tiller:$TAG_NAME
 
-    docker pull gcr.io/marketplace-k8s-app-tools/k8s/deployer_helm_tiller/onbuild:sha_$COMMIT_SHA
-    docker tag gcr.io/marketplace-k8s-app-tools/k8s/deployer_helm_tiller/onbuild:sha_$COMMIT_SHA gcr.io/$PROJECT_ID/k8s/deployer_helm_tiller/onbuild:$TAG_NAME
+    docker pull gcr.io/$_STAGING_PROJECT_ID/k8s/deployer_helm_tiller/onbuild:sha_$COMMIT_SHA
+    docker tag gcr.io/$_STAGING_PROJECT_ID/k8s/deployer_helm_tiller/onbuild:sha_$COMMIT_SHA gcr.io/$PROJECT_ID/k8s/deployer_helm_tiller/onbuild:$TAG_NAME
 
-    docker pull gcr.io/marketplace-k8s-app-tools/k8s/deployer_envsubst:sha_$COMMIT_SHA
-    docker tag gcr.io/marketplace-k8s-app-tools/k8s/deployer_envsubst:sha_$COMMIT_SHA gcr.io/$PROJECT_ID/k8s/deployer_envsubst:$TAG_NAME
+    docker pull gcr.io/$_STAGING_PROJECT_ID/k8s/deployer_envsubst:sha_$COMMIT_SHA
+    docker tag gcr.io/$_STAGING_PROJECT_ID/k8s/deployer_envsubst:sha_$COMMIT_SHA gcr.io/$PROJECT_ID/k8s/deployer_envsubst:$TAG_NAME
+
+- id: &DetermineShouldTagLatest Determine if images should be tagged latest
+  name: gcr.io/cloud-builders/gcloud
+  waitFor:
+  - *PublishImages
+  entrypoint: /bin/bash
+  args:
+  - -ceu
+  - echo "$(.cloudbuild/should_tag_latest.sh gcr.io/$PROJECT_ID/k8s/dev $TAG_NAME)" > should_tag_latest
+
+- id: Maybe tag images latest
+  name: gcr.io/cloud-builders/docker
+  wairFor:
+  - *DetermineShouldTagLatest
+  entrypoint: /bin/bash
+  args:
+  - -ceu
+  - |
+    if [[ "$(cat should_tag_latest)" == "true" ]]; then
+      docker tag gcr.io/$PROJECT_ID/k8s/dev:$TAG_NAME gcr.io/$PROJECT_ID/k8s/dev:latest
+      docker tag gcr.io/$PROJECT_ID/k8s/deployer_helm:$TAG_NAME gcr.io/$PROJECT_ID/k8s/deployer_helm:latest
+      docker tag gcr.io/$PROJECT_ID/k8s/deployer_helm/onbuild:$TAG_NAME gcr.io/$PROJECT_ID/k8s/deployer_helm/onbuild:latest
+      docker tag gcr.io/$PROJECT_ID/k8s/deployer_helm_tiller:$TAG_NAME gcr.io/$PROJECT_ID/k8s/deployer_helm_tiller:latest
+      docker tag gcr.io/$PROJECT_ID/k8s/deployer_helm_tiller/onbuild:$TAG_NAME gcr.io/$PROJECT_ID/k8s/deployer_helm_tiller/onbuild:latest
+      docker tag gcr.io/$PROJECT_ID/k8s/deployer_envsubst:$TAG_NAME gcr.io/$PROJECT_ID/k8s/deployer_envsubst:latest
```

Comment:
I have a comment here to discuss with you offline.

Reply:
Sync'd offline - need to invoke `docker push`.

We can revisit and clean up later.